### PR TITLE
[8.0] Some improvements for acsone public module template

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*~
+*.pyc
```
Comment (sbidoul) [inline .gitignore:2]: gitignore is usually put at the repository level, and not in each module, so I've doubts about this. Beside, there is much more complete version in OCA/MQT.

**Comment by sbidoul** [comment]
A doubt about gitignore, otherwise :+1:

File: .gitignore (hunk 1 above)
Comment (adrienpeiffer) [inline .gitignore:2]: Hi Stéphane,
The commit add gitignore at the repository level. it's not useful for the template. I should have done it in another PR. Thanks for the review !

Comment (sbidoul) [inline .gitignore:2]: Right! I had not seen that. Thanks!

**Comment by lmignon** [comment]
controllers should be also moved in a controller package.  What do you think?

**Comment by lmignon** [comment]
Otherwise :+1:
